src/spectral/extension.ts

```diff
@@ -42,6 +43,7 @@ export class SpectralExtension {
   constructor() {
     this.contextService = ContextService.getInstance()
     this.logger = LoggerService.getInstance()
+    Configuration.getInstance(workspace)
```

Comment:
Please use a class property and assign the instance to this prop.

Reply:
fixed, https://github.com/SpectralOps/vscode-extension/pull/7/commits/e1fded33e247a3fd2e4a7e247f1b8c1868623af6